I'm getting
```
/home/runner/work/skyline/skyline/app/src/main/cpp/skyline/gpu.cpp:403:11: error: no matching constructor for initialization of 'std::optional<ShaderManager>'
          shader(state, *this),
          ^      ~~~~~~~~~~~~
```
but I'm not sure if that is a code problem or a me problem.

--- a/app/src/main/cpp/skyline/gpu.cpp
+++ b/app/src/main/cpp/skyline/gpu.cpp
@@ -400,7 +400,6 @@ namespace skyline::gpu {
           buffer(*this),
           megaBufferAllocator(*this),
           descriptor(*this),
-          shader(state, *this),
           helperShaders(*this, state.os->assetFileSystem),
           graphicsPipelineCache(*this),
           renderPassCache(*this),

